python/pyspark/testing/connectutils.py

```diff
@@ -58,7 +63,23 @@ class PlanOnlyTestFixture(unittest.TestCase):
 
     connect: "MockRemoteSession"
 
+    @classmethod
+    def _read_table(cls, table_name: str) -> "DataFrame":
+        return DataFrame.withPlan(Read(table_name), cls.connect)  # type: ignore
+
+    @classmethod
+    def _udf_mock(cls, *args, **kwargs):
```

Comment:
nit:

```suggestion
    def _udf_mock(cls, *args, **kwargs) -> str:
```

Reply:
done. I thought `mypy` checks such case but it seems not to be true on this line.